### ISSUE TITLE
Todo: Allow using local assets

### ISSUE THREAD

**Issue by nomi-san** [nomi-san/league-loader#9]
Hosting assets such as CSSs, fonts, images and media on server takes a lot of time.
Now you can import them on local filesystem without any cost.

- [x] HTTPS based scheme: `//assets/` `https://assets/`
- [x] Allow query params

```html
<link rel="stylesheet/css" href="//assets/theme.css" />
<img src="//assets/banner.png" />
<video src="//assets/animated.webm" autoplay></video>
```

```
assets/
  |__theme.css
  |__banner.png
  |__animated.webm
plugins/
  |...
```

To prevent caching, just add trailing query params:
```js
const url = `//assets/some-resource?v=${Date.now()}`;
```

**Comment by nomi-san** [comment]
ESM is also supported.

```js
// assets/a.js
import b from './b.js';
console.log(b);

// assets/b.js
export default { greeting: 'hello' };
```

```js
// console
(() => import('//assets/a.js'))();

// output: [object] { greeting: 'hello' }
```

DevTools' traffic:
```http
GET https://assets/a.js   -> 200 OK
GET https://assets/b.js   -> 200 OK
```

**Comment by PrincessAkira** [comment]
How am i supposed to download into it though. Like  apart from outside of Client. I havent found a way to access it at all unless the files already exist

**Comment by nomi-san** [comment]
This feature is available since v0.5a.
All custom assets must be placed in `assets` folder.
